Tests/PointFreeTests/__Snapshots__/DiscountsTests/testDiscounts_LoggedIn.1.Conn.txt

```diff
@@ -0,0 +1,2924 @@
+                                    Iceland
+                                  </option>
+                                  <option value="IN">
+                                    India
+                                  </option>
+                                  <option value="ID">
+                                    Indonesia
+                                  </option>
+                                  <option value="IR">
+                                    Iran, Islamic Republic of
+                                  </option>
+                                  <option value="IQ">
+                                    Iraq
+                                  </option>
+                                  <option value="IE">
+                                    Ireland
+                                  </option>
+                                  <option value="IM">
+                                    Isle of Man
+                                  </option>
+                                  <option value="IL">
+                                    Israel
+                                  </option>
+                                  <option value="IT">
+                                    Italy
+                                  </option>
+                                  <option value="JM">
+                                    Jamaica
+                                  </option>
+                                  <option value="JP">
+                                    Japan
+                                  </option>
+                                  <option value="JE">
+                                    Jersey
+                                  </option>
+                                  <option value="JO">
+                                    Jordan
+                                  </option>
+                                  <option value="KZ">
+                                    Kazakhstan
+                                  </option>
+                                  <option value="KE">
+                                    Kenya
+                                  </option>
+                                  <option value="KI">
+                                    Kiribati
+                                  </option>
+                                  <option value="KP">
+                                    Korea, Democratic People&#39;s Republic of
+                                  </option>
+                                  <option value="KR">
+                                    Korea, Republic of
+                                  </option>
+                                  <option value="KW">
+                                    Kuwait
+                                  </option>
+                                  <option value="KG">
+                                    Kyrgyzstan
+                                  </option>
+                                  <option valu
```

Comment:
this test snapshot is for when you are logged in on the /discounts/{orgname} url, and here we can see the coupon code is exposed.